test/unit/command/test_idempotence.py

```diff
@@ -50,23 +50,25 @@ def test_execute_does_not_raise_on_converge_error(
 
 def test_execute_does_not_raise_on_idempotence_failure(
         patched_converge, patched_logger_error, molecule_instance):
-    patched_converge.return_value = None, 'changed=2'
+    output = 'check-command-01: ok=2    changed=1    unreachable=0    failed=0'
+    patched_converge.return_value = None, output
 
     i = idempotence.Idempotence({}, {}, molecule_instance)
     result = i.execute(exit=False)
 
-    msg = 'Idempotence test failed.'
+    msg = ''
```

Comment:
Not sure I understand why msg is being set to empty string and asserting it was called empty?